content/guides/authentication/index.md

```diff
@@ -18,7 +18,7 @@ console.
 | Box Application Type         | Authorization methods                                     |
 | ---------------------------- | --------------------------------------------------------- |
 | [Custom App][custom-app]     | [OAuth 2.0][oauth2] or [JWT][jwt]                         |
-| [Limited Access App]         | [App token][apptoken]                                     |
+|  Limited Access App          | [App token][apptoken]                                     |
```

Comment:
What are your thoughts on linking this to https://developer.box.com/guides/applications/select/#limited-access-app ? I can't find a better page where we could link to.